docs/topics/OAuth2.md

```diff
@@ -8,11 +8,11 @@ The first step in implementing OAuth2 is [registering a developer application](#
 
 ###### OAuth2 URLs
 
-| URL                                            | Description            |
-| ---------------------------------------------- | ---------------------- |
-| https://discord.com/api/oauth2/authorize    | Base authorization URL |
-| https://discord.com/api/oauth2/token        | Token URL              |
-| https://discord.com/api/oauth2/token/revoke | Revocation URL         |
+| URL                                            | Description                                                          |
+| ---------------------------------------------- | -------------------------------------------------------------------- |
+| https://discord.com/api/oauth2/authorize       | Base authorization URL                                               |
+| https://discord.com/api/oauth2/token           | Token URL                                                            |
+| https://discord.com/api/oauth2/token/revoke    | Revocation URL (see [RFC 7009](https://tools.ietf.org/html/rfc7009)) |
```

Comment:
imo the hyperlink should be like this: [Revocation URL](https://tools.ietf.org/html/rfc7009)

that way it provides details on it, but doesn't clutter up the space with a name (which isn't that useful if you have a link).

Other than that, lgtm!

Reply:
I changed it to `[Token Revocation](https://tools.ietf.org/html/rfc7009) URL` as I wanted the link to refer to the act of Token Revocation, rather than imply that it would link to info about how to use the URL.

I realize that might sound a bit nitpicky, but eh.